### ISSUE TITLE
Training a Neural Network in top

### ISSUE THREAD

**Issue by chinmay5** [facebookresearch/LASER#75]
Hi, I am bit of a noivce in the field so many of the things might seem trivial but I would appreciate some help in here. I am  trying to perform a product category identification on a multilingual dataset (there are statements about a product and the network needs to guess its category). For this :

1. Use LASER to get sentence embeddings
2. Train a FC Network on the 1024 dimensional output to scale it a bit
3. Train a softmax at the end
However, I was thinking if it would make sense to start using convolutions on the 1024 dimensional output vector from LASER? I was thinking on using things like a RESNET block with 1-D convolutions but I am not sure what is the implication of that. 
Frankly, we would be using such convolution mostly in language modeling. Is it even correct to try and use anything other than a FC layer?

I would really appreciate a response since I am basically stuck

**Comment by hoschwenk** [comment]
Hello,
it looks like that you can adapt the tool $LASER/source/sent_classif.py
It's used in the MLDoc task, i.e. classification of sentences into 4 categories. It uses an MLP classifier with potentially several hidden layers. You may try more sophisticated classifiers (with convolutions), but I'm not sure whether this is needed.
Hope this helps !
